src/Encrypt/Openssl.php

```diff
@@ -364,11 +364,17 @@ public function encrypt($value)
             $value    = $compress($value);
         }
 
-        $crypt  = openssl_seal($value, $encrypted, $encryptedkeys, $keys);
-        foreach ($keys as $key) {
-            openssl_free_key($key);
+        $crypt  = openssl_seal($value, $encrypted, $encryptedkeys, $keys, 'RC4');
+
+
+        // PHP 8 automatically frees the key instance and deprecates the function
+        if (PHP_VERSION_ID < 80000) {
+            foreach ($keys as $key) {
+                openssl_free_key($key);
+            }
         }
```

Comment:
The new method should also be used here.

Reply:
I left it as micro optimization as it does not require to execute `foreach` block. Do you suggest moving this `foreach` to that method?